src/definitions/behaviors/form.js

```diff
@@ -345,14 +345,14 @@
                     },
                     fieldDirty: function ($el) {
                         var initialValue = $el.data(metadata.defaultValue);
-                        // Explicitly check for null/undefined here as value may be `false`, so ($el.data(dataInitialValue) || '') would not work
-                        if (initialValue == null) {
+                        // Explicitly check for undefined/null here as value may be `false`, so ($el.data(dataInitialValue) || '') would not work
+                        if (initialValue === undefined || initialValue === null) {
                             initialValue = '';
                         } else if (Array.isArray(initialValue)) {
                             initialValue = initialValue.toString();
                         }
                         var currentValue = $el.val();
-                        if (currentValue == null) {
+                        if (currentValue === undefined) {
```

Comment:
https://api.jquery.com/val/

jQuery 3.0.0 (or higher) is required, it was released in 2016, is a minimal version of jQuery documented somewhere in the docs?

Reply:
🤔 The docs examples always show jquery 3.x , but infact it seems we havent explicitely told anywhere that jquery 3+ is needed.
only the package.json "says" you need jquery 3.4+
Regarding this LOC, the check for null would still fetch jquery 2, as jq3 will always return an array, so the next else branch would fetch it. Checking for undefined will never be reached it seems

Reply:
I think we should replace this by 
```suggestion
                        if (currentValue === null || currentValue === undefined) {
```
Doesn't hurt, does it? What do you think?

Reply:
> Doesn't hurt

when we need to support jQuery 1.x/2.x, it is needed :)